library/ziti.c

```diff
@@ -22,8 +22,22 @@ limitations under the License.
 #include <uv.h>
 #include "utils.h"
 #include "zt_internal.h"
+#include "ziti_enroll.h"
 #include <http_parser.h>
 
+
```

Comment:
can split up all the new functionality into separate files please
pkcs7, ziti_enroll, etc

Reply:
done

Reply:
unused includes `<mbedtls/*.h>`?

Reply:
I removed the obsolete includes